packages/Webkul/Shop/src/Resources/views/customers/account/orders/view.blade.php

```diff
@@ -33,6 +33,8 @@ class="grid md:hidden"
             </div>
 
             <div class="flex gap-1.5">
+                {!! view_render_event('bagisto.shop.customers.account.orders.reorder-btn.before') !!}
```

Comment:
Please use snake case for the event name
 {!! view_render_event('bagisto.shop.customers.account.orders.reorder_btn.before') !!}